common/script/index.js

```diff
@@ -2281,13 +2281,14 @@ api.wrap = function(user, main) {
           user.stats.lvl++;
           tnl = api.tnl(user.stats.lvl);
           user.stats.hp = 50;
-          if (user.stats.lvl > api.maxLevel) {
+          var totalStatPoints = user.stats.str + user.stats.int + user.stats.con + user.stats.per;
+          if (totalStatPoints >= api.maxLevel) {
```

Comment:
I think it is a constant: `api.maxLevel = statHelpers.MAX_LEVEL;` 
https://github.com/TheHollidayInn/habitrpg/blob/leveling-ensure-user-gets-100-points/common/script/index.js#L23